site/source/docs/optimizing/Optimizing-Code.rst

```diff
@@ -119,6 +119,59 @@ linker can handle a mix wasm object files and LTO object files.  Passing
 Thus, to allow maximal LTO opportunities with the LLVM wasm backend, build all
 source files with ``-flto`` and also link with ``flto``.
 
+EVAL_CTORS
+==========
+
+Building with ``-s EVAL_CTORS`` will evaluate as much code as possible at
```

Comment:
I think we should skip the space after `-s` these day in our docs.  I know we are not consistent but its more concise and compatible (especially with cmake) so I think we should prefer it.

Reply:
607 912 998